conntrack_linux.go

```diff
@@ -196,10 +204,11 @@ type ProtoInfo interface {
 type ProtoInfoTCP struct {
 	State uint8
 }
+
 // Protocol returns "tcp".
-func (*ProtoInfoTCP) Protocol() string {return "tcp"}
+func (*ProtoInfoTCP) Protocol() string { return "tcp" }
 func (p *ProtoInfoTCP) toNlData() ([]*nl.RtAttr, error) {
-	ctProtoInfo := nl.NewRtAttr(unix.NLA_F_NESTED | nl.CTA_PROTOINFO, []byte{})
+	ctProtoInfo := nl.NewRtAttr(unix.NLA_F_NESTED|nl.CTA_PROTOINFO, []byte{})
```

Comment:
oh! just now notice the formatting changes (I had to push and jump in a meeting); probably my IDE doing a `gofumpt` 😞 

let me know if you prefer having this as a separate commit 😅

Reply:
I reverted the formatting changes; will open a separate PR to do a format of the code.